backend/src/components/application.js

```diff
@@ -488,9 +488,10 @@ async function printPdf(req, numOfRetries = 0) {
 
     if (numOfRetries >= 5) {
       log.error(`printPdf :: Maximum retries reached for applicationId=${req.params.applicationId}. Aborting.`);
+      closeBrowser();
```

Comment:
This will log a warning if there are other browser contexts still open (multiple print jobs at once) on the pod.  Maybe use `await browserContext.close()` if it didn't get called on 479?

Reply:
That already happens on line 487. I think we're trying to close the loop where closeBrowser() (which normally happens on line 480) isn't called if the pdf never gets printed even after 5 retries.

Reply:
Ok fair enough.